lib/plugins/aws/deployFunction/index.test.js

```diff
@@ -139,5 +139,24 @@ describe('AwsDeployFunction', () => {
         fs.statSync.restore();
       });
     });
+
+    it('should read the provided artifact', () => {
+      sinon.stub(fs, 'readFileSync').returns('some dummy data');
```

Comment:
All the stubs should be moved in a `beforeEach()` block and reset in an `afterEach()` block.